cmux-tui/scripts/test_verify_published_manifest.py

```
@@ -0,0 +1,160 @@
+        before_latest_run = steps[before_latest]["run"]
+        self.assertEqual(
+            steps[before_latest]["env"]["EXPECTED_WINDOWS_ARTIFACT"],
+            self.WINDOWS,
+        )
+        self.assertIn("$EXPECTED_WINDOWS_ARTIFACT", before_latest_run)
+        after_publish_run = steps[after_publish]["run"]
+        self.assertEqual(
+            steps[after_publish]["env"]["EXPECTED_WINDOWS_ARTIFACT"],
+            self.WINDOWS,
+        )
+        self.assertIn(f"cmux-tui/$GITHUB_SHA/manifest.json", after_publish_run)
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)